### PR TITLE
Follow API changes for CredentialsController

### PR DESCRIPTION
**Pull request by CChemin**
Signed-off-by: CChemin <cecile.chemin@insee.fr>

### CODE REVIEW

File: src/api/remote/credentialApi.ts [hunk 1]
```diff
@@ -32,11 +32,11 @@ export const resetPasswordUs = (
...
 				'/users/' +
 				userId +
-				'/reinitPassword',
-			pcr,
+				'/reinit-password',
+			{ templateProperties: templateProperties },
 			{
 				params: {
-					sendModes: 'MAIL',
+					'webhook-tag': 'MAIL',
```
Comment (Donatien26) [inline src/api/remote/credentialApi.ts:39]: Maybe in the ui-conf becaus everybody will not have a webhook called mail ?

**Comment by Donatien26** [comment]
I don't know if it's possible to grep from api the template of the mail from the api and showing it completed unstead of hardcoding in ui the template ?

**Comment by Donatien26** [comment]
Maybe remove draft ? 0:)

**Comment by CChemin** [comment]
Yes I completely agree with you that a endpoint on the api showing the template (or maybe a location where both api and ui can get the template? ) would be way better than what is done here where the template displayed may no be consistent with the true template. Since it is not yet available I let it as it is.

**Review by clement-dufaure — APPROVED** [review]

**Pull request merged**